.gitignore

```diff
@@ -3,6 +3,7 @@
 .vscode
 .idea
 *~
+*.swp
```

Comment:
🤷 why not

Reply:
currently, i always need to close vim before doing `git add --a` to avoid the *.swps